examples/basic-bidding-sale/basic-bidding-sale.repl

```diff
@@ -190,23 +214,43 @@
 
   (env-data {
     "bid-id": (get-bid-id "DKc5HEWcmP8iPWue2WvMcnjn_WnowmuHz5Ogukm2SWk" "k:bidder")
-    ,"market-guard": {"keys": ["market"], "pred": "keys-all"}
-    ,"seller-guard": {"keys": ["account"], "pred": "keys-all"}
     ,"sale-id": "DKc5HEWcmP8iPWue2WvMcnjn_WnowmuHz5Ogukm2SWk"
   })
 
-  (marmalade-v2.abc.create-account "k:market" (read-keyset 'market-guard))
-
   (env-sigs
-  [{'key:'account
-    ,'caps: [
-      (UPDATE_QUOTE_GUARD (read-msg 'sale-id))
-      (SELLER (read-msg 'sale-id))
-    ]}
+    [{'key:'account
+      ,'caps: []}
   ])
 
   (expect "Accepting bid succeeds"
     true
     (accept-bid (read-msg "bid-id") (read-msg "sale-id"))
   )
 (commit-tx)
+
+(begin-tx "Finalize sale")
+  (env-hash (hash "finalize-sale-0"))
+  (use marmalade-v2.quote-manager)
+  (use marmalade-v2.ledger)
+  (use free.basic-bidding-sale)
+
+  (env-data {
+    "token-id": (create-token-id { 'uri: "test-uri", 'precision: 0, 'policies: [] } )
+    ,"buyer": "k:bidder"
+    ,"buyer-guard": {"keys": ["bidder"], "pred": "keys-all"}
+    ,"buyer_fungible_account": "k:bidder"
+    ,"sale-id": "DKc5HEWcmP8iPWue2WvMcnjn_WnowmuHz5Ogukm2SWk"
+  })
+
+  (env-sigs [
```

Comment:
In our sale flow, we shouldn't require the `bidder` to sign the `buy` step. They need to sign when they pay for the fungible. 
But we still need a signature, because `BUY` is a managed capability and needs to be in scope. 

We can test it with a `any` key and see if it still succeeds.

Reply:
Yes, good spot. I've updated the key